build-w-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Lines added to get the script running in the script path shell context
+# reference: http://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
+cd $(dirname $0)
+
+# To avoid issues with MINGW y Git Bash, see:
+# https://github.com/docker/toolbox/issues/673
+# https://gist.github.com/borekb/cb1536a3685ca6fc0ad9a028e6a959e3
+export MSYS_NO_PATHCONV=1
+export MSYS2_ARG_CONV_EXCL="*"
+
+rm -rf dev
+rm -rf dist
+
+docker build --pull -t doppler-ui-system-source .
+docker run --rm \
+    -v `pwd`/dist:/work/dist \
+    -p 3500:3500 \
+    -e CHOKIDAR_USEPOLLING=true \
```

Comment:
why are you using this `CHOKIDAR_USEPOLLING=true \` in the build script? As I undestand this is used to poll for watch purposes.

Reply:
It is necessary to start the virtual containers, this activates the updates when changes are made to the files.

Reply:
Both of you are fine. But, with a minor detail: it is necessary **for our watch** in the virtual container.

It is because _watch_ need to detect when you made changes, and with this variable, we are forcing use of a polling mechanism.

So, in this case, since we are not watching, please remove it.